src/public/parts/shells/chat/public/src/templates/hub/status/menu.html

```
@@ -1,6 +1,6 @@
-<ul class="menu menu-sm bg-base-100 rounded-box shadow-lg border border-base-300 p-1 z-50" style="${style}">
+<ul class="status-menu menu menu-sm bg-base-100 rounded-box shadow-lg border border-base-300 p-1 z-50" role="menu" data-i18n="chat.hub.changeStatusTitle" style="${style}">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)